pages/common/nrm.md

```diff
@@ -0,0 +1,27 @@
+# nrm
+
+> Nrm(npm registry manager) can help you easy and fast switch between different npm registries, now include: npm, cnpm, taobao, nj(nodejitsu), rednpm.
+
+- List all the registries:
+
+`nrm ls`
+
+- Change registry to registry:
+
+`nrm use {{registry}}`
+
+- Show the response time for all registries:
+
+`nrm test`
+
+- Add one custom registry:
+
+`nrm add {{registry}} {{url}}`
+
+- Delete one custom registry:
```

Comment:
Can this be used to delete only custom registries ? If not, then we should not use the word "custom".